helm/dagster/schema/schema_tests/test_dagit.py

```diff
@@ -0,0 +1,56 @@
+import pytest
+from kubernetes.client import models
+from schema.charts.dagster.subschema.dagit import Dagit, Server, Workspace
+from schema.charts.dagster.values import DagsterHelmValues
+from schema.charts.utils.kubernetes import Image, LivenessProbe, PullPolicy, Service, StartupProbe
+
+from .helm_template import HelmTemplate
+
+
+@pytest.fixture(name="template")
+def helm_template() -> HelmTemplate:
+    return HelmTemplate(
+        output="templates/deployment-dagit.yaml",
+        model=models.V1Deployment
+    )
+
+
+@pytest.mark.parametrize(
+    "service_port",
+    [
+        80,
+        3000,
+        8080,
+    ]
+)
+def test_dagit_port(template: HelmTemplate, service_port: int):
+    helm_values = DagsterHelmValues.construct(
+        dagit=Dagit(
+            service=Service(
+                type="ClusterIP",
+                port=service_port,
+            ),
+            image=Image(
+                repository="docker.io/dagster/dagster-celery-k8s",
+                tag="latest",
+                pullPolicy=PullPolicy.ALWAYS,
+            ),
+            workspace=Workspace(
+                enabled=False,
+                servers=[Server(host="k8s-example-user-code-1", port=3030)],
+            ),
+            livenessProbe=LivenessProbe(),
+            startupProbe=StartupProbe(),
+        )
+    )
```

Comment:
```suggestion
    helm_values = DagsterHelmValues.construct(
        dagit=Dagit.construct(
            service=Service(
                type="ClusterIP",
                port=service_port,
            ),
        )
    )
```

Reply:
We can use `.construct` in Pydantic to ignore other fields that are not relevant to the test. This way, we do not have to worry about editting the model. This will address your concern from https://github.com/dagster-io/dagster/pull/4062/files#r618851360